lib/screen/view_playlist/view_playlist.dart

```diff
@@ -0,0 +1,406 @@
+import 'package:autonomy_flutter/common/injector.dart';
+import 'package:autonomy_flutter/model/play_list_model.dart';
+import 'package:autonomy_flutter/model/sent_artwork.dart';
+import 'package:autonomy_flutter/screen/app_router.dart';
+import 'package:autonomy_flutter/screen/detail/artwork_detail_page.dart';
+import 'package:autonomy_flutter/screen/view_playlist/view_playlist_bloc.dart';
+import 'package:autonomy_flutter/screen/view_playlist/view_playlist_state.dart';
+import 'package:autonomy_flutter/service/settings_data_service.dart';
+import 'package:autonomy_flutter/util/constants.dart';
+import 'package:autonomy_flutter/util/play_control.dart';
+import 'package:autonomy_flutter/util/ui_helper.dart';
+import 'package:autonomy_flutter/view/artwork_common_widget.dart';
+import 'package:autonomy_flutter/view/responsive.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:nft_collection/models/asset_token.dart';
+import 'package:nft_collection/nft_collection.dart';
+import 'package:autonomy_flutter/service/account_service.dart';
+import 'package:autonomy_theme/autonomy_theme.dart';
+import 'package:autonomy_flutter/util/asset_token_ext.dart';
+import 'package:autonomy_flutter/service/configuration_service.dart';
+
+class ViewPlaylistScreen extends StatefulWidget {
```

Comment:
The `AddNewPlaylistScreen` and `ViewPlaylistScreen` look very similar. Can we do a refactor to reduce amount of duplicated code?